qa-jcoin.planx-pla.net/portal/gitops.json

```diff
@@ -724,5 +730,12 @@
     }
   ],
   "showArboristAuthzOnProfile": true,
-  "showFenceAuthzOnProfile": false
-}
\ No newline at end of file
+  "showFenceAuthzOnProfile": false,
+  "componentToResourceMapping": {
+    "Dashboard": {
+      "resource": "/secure_dashboard",
```

Comment:
the resource is `dashboard`
https://github.com/uc-cdis/commons-users/blob/506304c9130aec3b78916753e802321908ca16e0/users/qa-jcoin/user.yaml#L75